src/parsers/minecraft/list/lists.ts

```diff
@@ -0,0 +1,38 @@
+import { typed_keys } from "../../../misc-functions/third_party/typed-keys";
+
+export interface ListSupplier {
+    get(): string[];
+    init(): void;
+}
+
+const lists: { [key: string]: string } = {
+    "minecraft:color": "./lists/color",
+    "minecraft:entity_anchor": "./lists/entity_anchor",
+    "minecraft:item_enchantment": "./lists/enchantment",
+    "minecraft:item_slot": "./lists/item_slot",
+    "minecraft:mob_effect": "./lists/effect",
+    "minecraft:operation": "./lists/operation",
+    "minecraft:particle": "./lists/particle",
+    "minecraft:scoreboard_slot": "./lists/scoreboard_slot"
+};
+
+export class Lists {
+    protected suppliers: { [key: string]: ListSupplier };
+
+    public constructor() {
+        this.suppliers = {};
+    }
+
+    public getList(parserid: string): string[] {
+        return this.suppliers[parserid].get();
+    }
+
+    public registerLists(): void {
+        this.suppliers = {};
+        for (const s of typed_keys(lists)) {
+            const supplier = require(`./lists/${s}`) as ListSupplier;
```

Comment:
How does this work. The path `src/parsers/minecraft/list/lists.ts/./lists/minecraft:color` normalises to `src/parsers/minecraft/list/lists.ts/lists/minecraft:color`, which doesn't exist. Did you mean `require(lists[s])`?

Also, the variable name should probably be `list` or something